contracts/storage/VoteLockStorage.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.7.1;
+pragma experimental ABIEncoderV2;
+
+contract VoteLockStorage {
+    bytes32 constant DIAMOND_STORAGE_POSITION = keccak256("com.barnbridge.dao.votelock");
```

Comment:
should we name it vote lock storage position and voteLockStorage below the method and the struct? also

Reply:
Changed it, good catch!